commit 0e151ea2ab1663f3649688139739712060ae3259
Author: John Tapsell <johnflux@gmail.com>
Date:   Thu Apr 7 10:59:20 2011 +0100

    Fix accidental removal of a #include, when adding freebsd support.  This broke the lmsensors at minimum
    
    BUG: 270246
    BUG: 270246
    CCMAIL: naylor.b.david@gmail.com

diff --git a/ksysguard/ksysguardd/modules.h b/ksysguard/ksysguardd/modules.h
index e780be6..9aae890 100644
--- a/ksysguard/ksysguardd/modules.h
+++ b/ksysguard/ksysguardd/modules.h
@@ -23,6 +23,7 @@
 #ifndef KSG_MODULES_H
 #define KSG_MODULES_H
 
+#include "config-ksysguardd.h"
 #include "Command.h"
 #include "conf.h"
 #include "ksysguardd.h"
